core/pom.xml

```diff
@@ -156,10 +156,6 @@ THE SOFTWARE.
       <groupId>commons-codec</groupId>
       <artifactId>commons-codec</artifactId>
     </dependency>
-    <dependency>
-      <groupId>commons-collections</groupId>
-      <artifactId>commons-collections</artifactId>
-    </dependency>
```

Comment:
Before this can be removed, we should identify usages in plugins. See [this guide](https://www.jenkins.io/doc/developer/plugin-development/usage-in-plugins/) for how to do that.